styles/style.scss

```diff
@@ -48,13 +48,18 @@ h2 {
     font-family: $heading-font;
     font-weight: 700;
     margin-bottom: 2%;
+    text-align: left;
```

Comment:
were all `h2`'s on the site not already left-aligned?

Reply:
oops I think there was a centertext class added before, I forgot to delete the text-align after I realized